### ISSUE TITLE
Closing ffmpeg when isn't used

### ISSUE THREAD

**Issue by rudnypc** [m1k1o/go-transcode#25]
The ffmpeg will be killed after 50 segments are created.
But When I start a video with a index file, the vlc start all playlists and some ffmpeg instances are started, this slows down the server, maybe there should be a better control to disable ffmpeg when not used, and when used, increase the limit to 50 so there is no cut in audio between old and new segments.

**Comment by rudnypc** [comment]
When the ffmpeg is closed, is the last segment complete?
When I try to kill the ffmpeg, always the last segment is corrupt.

**Comment by m1k1o** [comment]
Yes, it does not kill ffmpeg just waits it to finish. That would need to be implemented. What kill signal do you use?

**Comment by rudnypc** [comment]
I tried "kill -9|HUP|SIGINT", pressing "q"... I saw the ffmpeg command with only "-ss", because this I asked about it.

**Comment by rudnypc** [comment]
I have tested ffmpeg a lot, the best video process is to keep ffmpeg until the end of the video when getting new segment, you can close ffmpeg only if it asks for a future segment (seek) or if it doesn't request a new segment for a while (timeout ).
